tests/framework/clients/dynamic/dynamic.go

```diff
@@ -23,7 +23,7 @@ type Client struct {
 
 // NewForConfig creates a new dynamic client or returns an error.
 func NewForConfig(ts *session.Session, inConfig *rest.Config) (dynamic.Interface, error) {
-	logrus.Infof("Dynamic Client Host:%s", inConfig.Host)
+	logrus.Debugf("Dynamic Client Host:%s", inConfig.Host)
```

Comment:
This log is very spammy during integration tests. I figure it should be a debug log.